tests/factories/test_service.py

```diff
@@ -65,3 +65,7 @@ class PermissionPolicy(RecordPermissionPolicy):
     res = service.search(identity_simple, q=f"id:{id_}", size=25, page=1)
     assert res.total == 1
     assert list(res.hits)[0] == read_item.data
+
+    # Check to see if the community exists
+    assert service.exists(identity_simple, id_) == True
+    assert service.exists(identity_simple, "not-a-valid-id") == False
```

Comment:
could we add a test case checking if permissions are also respected? i.e. if the user who does not have permissions to read the record tries to check if it exists

Reply:
Sure, I'll give it a go